conda/plugins/manager.py

```diff
@@ -78,11 +78,17 @@ class CondaPluginManager(pluggy.PluginManager):
     The conda plugin manager to implement behavior additional to pluggy's default plugin manager.
     """
 
-    #: Cached version of the :meth:`~conda.plugins.manager.CondaPluginManager.get_solver_backend`
+    get_cached_solver_backend: Callable[[str | None], type[Solver]]
```

Comment:
How did I loose the docstring?

```suggestion
    #: Cached version of the :meth:`~conda.plugins.manager.CondaPluginManager.get_solver_backend`
    #: method.
    get_cached_solver_backend: Callable[[str | None], type[Solver]]
```